spec/controllers/gallery/gallery_controller_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe GalleryController, type: :controller do
+  let(:user) { create(:user) }
+  let(:album) { create(:album) }
+  let(:old_album) { create(:album, start_date: Time.zone.parse("1986-03-25T12:00:00")) }
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.